Assets/Scripts/Combat/Enemy.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Enemy : Entity
+{
+    public override void Respawn() {
+    }
+
+    public override void Die() {
+        Debug.Log("Enemy died!");
+
+        this.enabled = false;
```

Comment:
This will actually disable the actual script itself and not the object it belongs to. If you want the enemy to disappear completely from the game you can call `Destroy(gameObject);` which will remove the gameobject from the scene.